cmd/main.go

```diff
@@ -38,12 +37,13 @@ func main() {
 	todoHTTP := todohttp.New(todoSvc, app.Logger)
 
 	public := http.FileServer(http.Dir("public"))
-	swagger := http.FileServer(http.Dir("openapi"))
+	openapi := http.FileServer(http.Dir("openapi"))
 
 	http.Handle("/public/", http.StripPrefix("/public/", public))
-	http.Handle("/swagger/", http.StripPrefix("/swagger/", swagger))
+	http.Handle("/openapi/", http.StripPrefix("/openapi/", openapi))
 
 	http.HandleFunc("/", server.Chain(todoHTTP.Root, server.Method(http.MethodGet)))
+	http.Handle("/swagger/", http.StripPrefix("/swagger/", server.Chain(todoHTTP.Swagger, server.Method(http.MethodGet))))
```

Comment:
this endpoint is not working

Reply:
changed the endpoint to `/api` and also make it in page query param as `/?page=api`